scripts/travis/minikube.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+echo "==============================================================================================================="
+echo " minikube setup"
+echo "==============================================================================================================="
+
+curl -Lo minikube https://storage.googleapis.com/minikube/releases/v1.8.1/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
+mkdir -p $HOME/.kube $HOME/.minikube
+touch $KUBECONFIG
```

Comment:
where is `KUBECONFIG` defined? and whats the default value.

Reply:
```
 CHANGE_MINIKUBE_NONE_USER=true
 MINIKUBE_WANTUPDATENOTIFICATION=false
 MINIKUBE_WANTREPORTERRORPROMPT=false
 MINIKUBE_HOME=$HOME
 CHANGE_MINIKUBE_NONE_USER=true
 KUBECONFIG=$HOME/.kube/config
```
These are global vars declared in travis

Reply:
hmm do we need those in travis settings, these will be common to all right. we can put in here in the script.